File: maximum_minimum_elements_in_an_Array.py
def find_maxi(arr):
    maxi = max(arr)
    return maxi

def find_mini(arr):
    mini = min(arr)
    return mini


n = int(input("Enter the number of elements you want to store in an array : "))
arr = []
for i in range (n):
    arr.append(int(input(f"Enter the element {i+1} : ")))
print("The maximum element in an array : ",find_maxi(arr))
print("The minimum element in an array : ",find_mini(arr))

File: maximum_minimum_elements_in_an_Array.cpp
#include<iostream>

using namespace std;
int find_maxi(int arr[],int n)
{
    int max = INT32_MIN;
    for(int i = 0;i<n;i++)
    {
        if(max <= arr[i])
        {
            max = arr[i];
        }
    }
    return max;
}
int find_mini(int arr[],int n)
{
    int min = INT32_MAX;
    for(int i = 0;i<n;i++)
    {
        if(min >= arr[i])
        {
            min = arr[i];
        }
    }
    return min;

}
int main(){
int n;
cin >> n;
int arr[n];
for(int i = 0;i<n;i++)
{
    cin >> arr[i];
}
cout << "The maximum element in an array is : "<<find_maxi(arr,n)<<endl;
cout << "The minimum element in an array is : "<<find_mini(arr,n)<<endl;
return 0;
}